### DISCUSSION TITLE
Pause Menu

### DISCUSSION THREAD

**Discussion by NavySaw23** [anshikasrivastava17/Snakes-Project#6] [Ideas]
A really fun game to play, Great job on the design as well. I however have a suggestion to add a pause menu. As the game speeds up and gets faster, it's a good idea to let the user have some time to do some wrist exercises with the help of a pause menu.

**Comment by anshikasrivastava17** [comment]
Thankyou for such a useful and practical suggestion. Will surely try implementing it.
